src/turndown.js

```diff
@@ -348,6 +348,29 @@ turndownService.addRule("edu_breakdown", {
   }
 })
 
+/**
+ * Catch "approx" images and render as HTML instead
+ */
+turndownService.addRule("approx_img", {
+  filter: (node, options) => {
+    if (
+      node.nodeName === "IMG" &&
+      node.getAttribute("src").includes("-approx.png")
+    ) {
+      return true
+    }
+    return false
+  },
+  replacement: (content, node, options) => {
+    const matches =
+      node.getAttribute("src").match(/(\/|_)[0-9]+-approx.png/g) || []
+    if (matches.length > 0) {
+      const amount = matches[0].substring(1).replace("-approx.png", "")
```

Comment:
We should use a named group to capture the part of the string that we want. For example:

    const match = node.getAttribute("src").match(/(\/|_)(?<amount>[0-9]+)-approx.png/)
    if (match) {
      const amount = match.groups.amount
      return `{{< approx-students ${amount} >}}`
    }